README.md

```diff
@@ -47,14 +67,16 @@ The **model.py** provides a scoring function with the following signature: `gene
 
 **Please note that in order to run the model API you need `ctranslate2 3.17.1` which is not part of the environment base mentioned in the section below. You will have to add this dependency to the environment Dockerfile instructions.** 
 
-## Set up instructions
 
 This project requires the following [compute environments](https://docs.dominodatalab.com/en/latest/user_guide/f51038/environments/) to be present. Please ensure the "Automatically make compatible with Domino" checkbox is selected while creating the environment.
 
-### Hardware Requirements
+If you are interested in running just the Jupyter notebooks, use a 
+
+### Hardware Requirements for Python File with API 
+
 You also need to make sure that the hardware tier running the notebook or the fine-tuning script has sufficient resources. A GPU with >=16GB of VRAM is recommended. This project was tested on a `V100` with **16GB** VRAM. Also note that the model binary occupies ~ **28GB** on disc so please provision your workspace volume accordingly.
 
-### Environment Requirements
+### Environment Requirements for Python File API 
 
```

Comment:
This is supposed to be about the dependencies needed in order to run the code. Since this template is used in Domino, usually this translates to the Domino Environment